neurox/interpretation/linear_probe.py

```diff
@@ -88,7 +88,9 @@ def _train_probe(
     tasks in order to train probes for them. A logistic regression model
     is trained with Cross Entropy loss for classification tasks and a linear
     regression model is trained with MSE loss for regression tasks. The
-    optimizer used is Adam with default ``torch.optim`` hyperparameters.
+    optimizer used is Adam with default ``torch.optim`` hyperparameters. 
+    The individual batches generated from the X_train inputs are converted to flaot32, such that
```

Comment:
typo "flaot32", break the line to roughly align with the text while you are fixing this